training/creating_dataset.py

```diff
@@ -30,17 +30,39 @@ def create_dataframe(
     :param characteristics: list of characteristics to load
     :return: DataFrame with monsters (NPC) form chosen books and with chosen characteristics and their origin book
     """
-    loading_methods = {
-        "system/abilities": move_values_level_up(value_name="mod"),
-        "system/attributes/ac": load_subcolumn_as_value("ac"),
-        "system/attributes/hp": load_subcolumn_as_value("hp"),
-    }
+
     for i in range(len(books) - 1, -1, -1):
         if not is_path_correct(books[i]):
             books.pop(i)
 
     bestiary = get_merged_bestiaries(books)
     bestiary = bestiary[bestiary["type"] == "npc"]
+
+    df = standardize_bestiary(bestiary, characteristics)
+
+    return df
+
+
+def standardize_bestiary(
+    bestiary: pd.DataFrame,
+    characteristics: list[str] = [
+        "system/abilities",
+        "system/attributes/ac",
+        "system/attributes/hp",
+    ],
```

Comment:
Typ mutowalny **nigdy** nie może być wartością domyślną w Pythonie! To wynika z tego, jak interpreter Pythona czyta moduły. Wartość domyślna jest tworzona po prostu jako wartość zmiennej `characteristics` przy pierwszym czytaniu (czyli imporcie) tego modułu. Jeżeli zostanie w jakimkolwiek momencie zmieniona, np. coś dopiszecie do tej listy, to to jest permanentne, następne wywołanie tej funkcji zobaczy inną wartość domyślną. Nawet, jak nie zmieniacie listy, to to jest po prostu zła praktyka. Zamiast tego robi się typ `Optional[list[str]]`, wartość domyślną `None`, i na początku funkcji to sprawdza - jak jest `None`, to przypisujemy wartość domyślną. Irytujące, ale tak działa Python